packages/server/lib/refreshTokens.ts

```diff
@@ -1,13 +1,13 @@
 import * as cron from 'node-cron';
-import { errorManager, ErrorSourceEnum, connectionService } from '@nangohq/shared';
+import { db, errorManager, ErrorSourceEnum, connectionService } from '@nangohq/shared';
 import { stringifyError, getLogger, metrics } from '@nangohq/utils';
 import tracer from 'dd-trace';
 import { logContextGetter } from '@nangohq/logs';
 
 const logger = getLogger('Server');
 
 export function refreshTokens(): void {
-    cron.schedule('0 6 * * *', () => {
+    cron.schedule('*/45 * * * *', async () => {
```

Comment:
I am curious to know why you picked 45 mins

Reply:
Arbitrarily. But it was a good middle ground between 30 and 1 hour. With 45 minutes it'll take about 2 days to go through the connections. I want to see how long an iteration takes and can tweak it further if needed.